src/Dialogs/Dialogs.vala

```diff
@@ -116,11 +114,15 @@ public File? choose_dir (string? user_title = null) {
         current_export_dir = File.new_for_path (Environment.get_home_dir ());
 
     if (user_title == null)
-        user_title = _ ("Export Photos");
-
-    Gtk.FileChooserDialog chooser = new Gtk.FileChooserDialog (user_title,
-            AppWindow.get_instance (), Gtk.FileChooserAction.SELECT_FOLDER, _ ("_Cancel"),
-            Gtk.ResponseType.CANCEL, _ ("_Select"), Gtk.ResponseType.ACCEPT, null);
+        user_title = _("Export Photos");
+
+    var chooser = new Gtk.FileChooserDialog (
```

Comment:
Should be Gtk.FileChooserNative  (else Cancel button missing)

Reply:
Oops, silly mistake